### ISSUE TITLE
Does not honor default shell

### ISSUE THREAD

**Issue by stovocor** [aarnt/octopi#92]
I am using zsh as my default shell. Octopi starts yaourt always with a bash, ignoring any settings (like EDITOR) which I set for zsh.

**Comment by EmbraceUnity** [comment]
I have the same problem.

**Comment by EmbraceUnity** [comment]
Oh nice, it was probably fixed 4 days ago with ad2246d440ea302159efd47647108b060fd02191

Not sure if this fixes it for yaourt too.

**Comment by aarnt** [comment]
Could you test current Git code to see if this issue is fixed?

**Comment by stovocor** [comment]
Replacing Octopi with a self-compiled version and downgrading an AUR package is a lot of work. I am sorry, but I do not have the time for it right now. If it is okay with you, I will test the next release and report here if it works.

**Comment by aarnt** [comment]
No problem, stovocor!

**Comment by EmbraceUnity** [comment]
I built it with the latest code. It works, but yaourt has its own /etc/yaourtrc file which controls the default editor. Thus, when I tried it, the $EDITOR variable in zshrc was not respected, and it prompted to provide the editor you wish to use, and gave an irrelevant message about setting bashrc.

It wasn't very hard for me to set the EDITOR and VISUAL variables inside of yaourtrc... so this is probably an upstream issue, if anything. A workaround would likely not be following the Arch Way.
